web-api/tests/Kerbero.Infrastructure.Tests/SmartLockKeys/Repositories/SmartLockKeyPersistentRepositoryTests.cs

```diff
@@ -0,0 +1,61 @@
+using FluentAssertions;
+using Kerbero.Domain.SmartLockKeys.Entities;
+using Kerbero.Infrastructure.Common.Context;
+using Kerbero.Infrastructure.Common.Interfaces;
+using Kerbero.Infrastructure.SmartLockKeys.Repositories;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Moq;
+
+namespace Kerbero.Infrastructure.Tests.SmartLockKeys.Repositories;
+
+public class SmartLockKeyPersistentRepositoryTests
+{
+	private readonly SmartLockKeyPersistentRepository _smartLockKeyPersistentRepository;
+
+	public SmartLockKeyPersistentRepositoryTests()
+	{
+		var options = new DbContextOptionsBuilder<ApplicationDbContext>()
+			.UseInMemoryDatabase(databaseName: "AppDbContext")
```

Comment:
using the same database name for every test is very bad, it may trigger failures on parallel tests